crates/router/src/routes/payments.rs

```diff
@@ -321,6 +322,14 @@ pub async fn payments_confirm(
         return http_not_implemented();
     };
 
+    if payload.customer_ip.is_none() {
+        payload.customer_ip = req
+            .headers()
+            .get("x-forwarded-for")
```

Comment:
add this string literal to `headers`.
https://github.com/juspay/hyperswitch/blob/6f1a1a3bcb7466738b018a773a5fe4adc5cf2887/crates/router/src/lib.rs#L47